packages/babel-helper-hoist-variables/src/index.ts

```diff
@@ -1,20 +1,37 @@
 import * as t from "@babel/types";
+import type { Identifier, VariableDeclaration } from "@babel/types";
```

Comment:
Nit, but maybe we could use `t.Identifier` and `t.VariableDeclaration` instead, since that's what we usually do in other packages?